apps/rush-lib/src/schemas/version-policies.schema.json

```diff
@@ -105,10 +109,14 @@
         "lockedMajor": {
           "description": "The locked major version",
           "type": "number"
+        },
```

Comment:
There is a somewhat wonky way to do inheritance in JSON schema. Perhaps you can dedupe this a little by doing that.

Reply:
Schema inheritance doesn't seem to support the "additionalProperties": false :(